examples/inverse/plot_dics_source_power.py

```diff
@@ -50,24 +51,19 @@
 # Read forward operator
 forward = mne.read_forward_solution(fname_fwd)
 
-# Computing the data and noise cross-spectral density matrices
-# The time-frequency window was chosen on the basis of spectrograms from
-# example time_frequency/plot_time_frequency.py
-# As fsum is False csd_epochs returns a list of CrossSpectralDensity
-# instances than can then be passed to dics_source_power
-data_csds = csd_epochs(epochs, mode='multitaper', tmin=0.04, tmax=0.15,
-                       fmin=15, fmax=30, fsum=False)
-noise_csds = csd_epochs(epochs, mode='multitaper', tmin=-0.11,
-                        tmax=-0.001, fmin=15, fmax=30, fsum=False)
+# Computing the cross-spectral density matrix at 6-10 Hz.
+# The frequency was chosen on the basis of spectrograms from
+# example time_frequency/plot_time_frequency.py.
+# We use a decim value of 20 to speed up the computation in this example at the
+# loss of accuracy.
+csd = csd_epochs(epochs, mode='cwt_morlet', tmin=0, tmax=0.5, decim=20,
+                 frequencies=np.linspace(6, 10, 4))
 
 # Compute DICS spatial filter and estimate source power
-stc = dics_source_power(epochs.info, forward, noise_csds, data_csds)
+filters = make_dics(epochs.info, forward, csd)
+stc, freqs = apply_dics_csd(csd, filters)
```

Comment:
why do you need to return freqs?

Reply:
The `stc` object stores the power map for each frequency along the "time" dimension and I'm thinking this is the best way to give the user access to the list of frequencies.

**I don't want to store it in** `stc.times`:  
As we saw in #4426, storing the frequencies in the `stc.times` attribute is not desirable and in this case even impossible, because there is no guarantee that the frequencies will be evenly spaced. Instead, `stc.times` is set to `np.arange(n_freqs)`.

**I don't want to rely on** `csd.frequencies`:
Of course, the frequencies can also be found in the `csd` object, but it is very convenient to do something like `make_dics(csd.mean())` and now the `csd.frequencies` no longer correspond to the `stc` frequencies. Having to keep the exact `csd` object around that was used in the power mapping operation seems more cumbersome to me then just keeping the `freqs` list around.

**It is consistent with how functions like** `pwelch` **return frequencies**:
Scipy functions that deal with frequencies also usually return `freqs` as a separate list. The pattern should be familiar to most users.